--- i18n.php.orig	2004-08-09 12:37:29.000000000 +0300
+++ i18n.php	2004-08-09 12:38:20.000000000 +0300
@@ -544,6 +544,10 @@
 $languages['he_IL']['DIR']     = 'rtl';
 $languages['he']['ALIAS']      = 'he_IL';
 
+$languages['ug']['NAME']    = 'Uighur';
+$languages['ug']['CHARSET'] = 'utf-8';
+$languages['ug']['DIR']     = 'rtl';
+
 /* Detect whether gettext is installed. */
 $gettext_flags = 0;
 if (function_exists('_')) {
